modules/admin/proc-running-rhdh-behind-a-proxy.adoc

```diff
@@ -0,0 +1,115 @@
+= Running the {product-very-short} application behind a corporate proxy
+
+You can run the {product-very-short} application behind a corporate proxy by setting any of the following environment variables before starting the application:
+
+* `HTTP_PROXY`: Denotes the proxy to use for HTTP requests.
+* `HTTPS_PROXY`: Denotes the proxy to use for HTTPS requests.
+
+Additionally, you can set the `NO_PROXY` environment variable to exclude certain domains from proxying. The variable value is a comma-separated list of hostnames that do not require a proxy to get reached, even if one is specified.
+
+For Helm-based deployment, both a developer or a cluster administrator, with permissions to create resources in the cluster, can configure the proxy variables in a `values.yaml` Helm configuration file.
+
+For Operator-based deployment, the approach you use for proxy configuration is based on your role:
+
+* As a cluster administrator with access to the Operator namespace, you can configure the proxy variables in the Operator's default ConfigMap file. This configuration applies the proxy settings to all the users within the namespace where the Operator is deployed.
+* As a developer, you can configure the proxy variables in a custom resource (CR) file. This configuration applies the proxy settings only to the users with access to that CR.
+
+
+.Prerequisites
+
+* You have installed the {product}. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+* You have created a project in the {product-short}.
+
+.Procedure
+
+. Configure proxy information in one of the following deployments based on your needs:
++
+.Helm deployment
+
+* Set the proxy information in your Helm configuration file:
++
+[source,yaml]
+----
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: HTTP_PROXY
+        value: '<http_proxy_url>'
+      - name: HTTPS_PROXY
+        value: '<https_proxy_url>'
+      - name: NO_PROXY
+        value: '<no_proxy_settings>'
+----
++
+Where,
+
+`<http_proxy_url>`:: Denotes a variable that you must replace with the HTTP proxy URL.
+`<https_proxy_url>`:: Denotes a variable that you must replace with the HTTPS proxy URL.
+`<no_proxy_settings>`:: Denotes a variable that you must replace with comma-separated URLs, which you want to exclude from proxying, for example, `foo.com,baz.com`.
++
+.Example: Setting proxy variables using Helm Chart
+
+[source,yaml]
+----
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: HTTP_PROXY
+        value: 'http://10.10.10.105:3128'
+      - name: HTTPS_PROXY
+        value: 'http://10.10.10.106:3128'
+      - name: NO_PROXY
+        value: 'localhost,example.org'
+----
+
++
+.Operator deployment 
+
+** Perform one of the following steps based on your role:
+* As an administrator, set the proxy information in the Operator's default ConfigMap file:
++
+.. Search for a ConfigMap file named `backstage-default-config` in the namespace where the Operator is deployed and open it. 
```

Comment:
```suggestion
.. Search for a ConfigMap file named `backstage-default-config` in the namespace where the Operator is deployed (`rhdh-operator` by default) and open it. 
```

The namespace where the RHDH operator is `rhdh-operator` by default. Should we add that here?

Reply:
Since we follow mod doc structure, we require to ensure that all the info we provide in a module is complete in itself. so, addint this info wud definitely be helpful.